motion/thruster_allocator/src/allocator_ros.cpp

```diff
@@ -88,31 +85,13 @@ void Allocator::callback(const geometry_msgs::Wrench &msg_in) const
   m_pub.publish(msg_out);
 }
 
-Eigen::VectorXd Allocator::rovForcesMsgToEigen(const geometry_msgs::Wrench &msg) const
+Eigen::VectorXd Allocator::bodyFrameForcesWrenchToEigen(const geometry_msgs::Wrench &msg) const
```

Comment:
Given the current implementation I think it makes sense to simply call this method "wrenchMsgToEigen()". The input is already assumed to be in BODY, so the method does not perform any frame transformation.